frontend/src/container/PipelinePage/tests/PipelinesSearchSection.test.tsx

```diff
@@ -20,4 +21,43 @@ describe('PipelinePage container test', () => {
 		);
 		expect(asFragment()).toMatchSnapshot();
 	});
+
+	it('should handle search', async () => {
+		const setPipelineValue = jest.fn();
+		const { getByPlaceholderText, container } = render(
+			<MemoryRouter>
+				<Provider store={store}>
+					<I18nextProvider i18n={i18n}>
+						<PipelinesSearchSection setPipelineSearchValue={setPipelineValue} />
+					</I18nextProvider>
+				</Provider>
+			</MemoryRouter>,
+		);
+
+		const searchInput = getByPlaceholderText('search_pipeline_placeholder');
+
+		// Type into the search input
+		userEvent.type(searchInput, 'sample_pipeline');
```

Comment:
we should add `await` for all the user events rather than awaiting for the result as in line 43

Reply:
Check again in coming PR